### PR TITLE
fix action description

### PR DESCRIPTION
**Pull request by skazi0**

### CODE REVIEW

**Comment by fichtner** [comment]
Hmm, correct but I am unsure why we need descriptions here anyway, maybe a historical oversight. Let me check when I have some time available...

**Comment by skazi0** [comment]
@fichtner Sure. I found this by looking at the cron options. There are two with the same description. I'm not sure if there's any use for this one in cron though. Feel free to change it to "remove action description" :o)

File: src/opnsense/service/conf/actions.d/actions_firmware.conf [hunk 1]
```diff
@@ -15,7 +15,7 @@ message:Fetching changelog from remote
 command: /usr/local/opnsense/scripts/firmware/changelog.sh remove
 parameters:
 type:script
-description:Firmware changelog update
+description:Firmware changelog remove
```
Comment (fichtner) [inline src/opnsense/service/conf/actions.d/actions_firmware.conf:18]:
```suggestion
```

Indeed it's better to drop this

**Comment by fichtner** [comment]
Nice catch, thanks!

**Pull request merged**